api/v1alpha1/envoyproxy_metric_types.go

```diff
@@ -29,6 +29,12 @@ type ProxyMetrics struct {
 
 	// EnableVirtualHostStats enables envoy stat metrics for virtual hosts.
 	EnableVirtualHostStats bool `json:"enableVirtualHostStats,omitempty"`
+
+	// EnablePerEndpointStats enables envoy stats metrics for peer endpoint
+	// WARNING: This field will let Envoy record the statistics of each endpoint.
```

Comment:
this rm L34 and L35

Reply:
a small question, why don’t we keep the warning information?